tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -0,0 +1,231 @@
+			const wpAdmin = new WpAdminPage( page, testInfo );
+			await wpAdmin.setExperiments( {
+				container: true,
+				'nested-elements': true,
+			} );
+
+			const editor = await wpAdmin.useElementorCleanPost(),
+				container = await editor.addElement( { elType: 'container' }, 'document' );
+
+			// Add widgets.
+			await editor.addWidget( 'nested-tabs', container );
+			await editor.getPreviewFrame().waitForSelector( '.e-n-tabs-content .e-con.e-active' );
+
+			// Set icons to tabs according 'TabsIcons' array.
+			await setIconsToTabs( page, TabsIcons );
+			const icon = await editor.getPreviewFrame().locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon' ).first(),
+				activeTabIcon = await editor.getPreviewFrame().locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon.e-active' ).first(),
+				currentContext = editor.getPreviewFrame();
+
+			// Assert
+			await page.locator( `.elementor-control-tabs_justify_horizontal [original-title='${ locationOption }']` ).first().click();
+			await expect( icon ).toHaveCSS( 'align-items', 'center' );
+			await clickTab( currentContext, '1' );
+			await expect( activeTabIcon ).toHaveCSS( 'align-items', 'center' );
+			await clickTab( currentContext, '0' );
+
+			await wpAdmin.setExperiments( {
+				container: false,
+				'nested-elements': false,
+			} );
+		} );
+	}
+} );
+
+test.describe( 'Nested Tabs - Inline font icons experiment', async () => {
+	test( `Icon should be visible when Inline font icons experiment is active`, async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+			'nested-elements': true,
+			e_font_icon_svg: true,
+		} );
+
+		await wpAdmin.openNewPage();
+
+		const editor = new EditorPage( page, testInfo ),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Add widgets.
+		await editor.addWidget( 'nested-tabs', container );
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tabs-content .e-con.e-active' );
+
+		// Set icons to tabs according 'TabsIcons' array.
+		await setIconsToTabs( page, TabsIcons );
+
+		await editor.publishAndViewPage();
+
+		await page.waitForSelector( '.elementor-widget-n-tabs' );
+
+		const icon = await page.locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon' ).first(),
+			activeTabIcon = await page.locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon.e-active' ).first(),
+			currentContext = page;
+
+		// Assert
+		await expect( activeTabIcon ).toBeVisible();
+		await clickTab( currentContext, '1' );
+		await expect( icon ).toBeVisible();
+		await clickTab( currentContext, '0' );
+
+		// Reset experiments.
+		await wpAdmin.setExperiments( {
+			container: false,
+			'nested-elements': false,
+			e_font_icon_svg: false,
+		} );
+	} );
+
+	test( `Should change icon size`, async ( { page }, testInfo ) => {
```

Comment:
For which task it this?

Reply:
The task was updated.
The test purpose is to check if the icon size is changing in the frontend.

Reply:
Test title updated by suggestion.